### DISCUSSION TITLE
How to update a setting?

### DISCUSSION THREAD

**Discussion by ElisameAraujo** [rawilk/laravel-settings#70] [Q&A]
Okay, I might have a dumb question but, is there a helper that I can use to update a setting?

For example, I use the method `Settings::set('foo', 'bar')` to create a setting in the DB, but how I do to update this setting if I need to in the future? I only see option to retrieve the settings, but not to update.

**Comment by rawilk** [comment]
Hey there, sorry I didn't see this until now. I'm sure you've figured this out by now, but you can use the `set()` method to both create and update a setting.
